src/index.js

```diff
@@ -60,6 +65,17 @@ const downloadImages = () => {
 };
 
 /**
+ * Make sure the filenames are safe by stripping out problematic characters
+ * @param {String} filename a potential filename
+ * @returns {String} filename with characters stripped out
+ */
+const cleanFileName = function (filename) {
```

Comment:
Do we really need this? I have never seen any problematic characters coming from the Tumblr API. I think this is a nice and smart function but not really sure if we really need it in this module. Can you please explain a little further about what was happening?